### ISSUE TITLE
Remote tunnels: Refused to set unsafe header User-Agent

### ISSUE THREAD

**Issue by connor4312** [microsoft/dev-tunnels#482]
> Testing [#5085](https://github.com/microsoft/vscode-internalbacklog/issues/5085)
> 
> 1. On web, try connecting to a remote tunnel
> 
> **Bug**
> 
> In the dev tools console logs I see the error:
> 
> ```
> Refused to set unsafe header "User-Agent"
> ```
> 
> Appears to come from this (minified) code
> 
> ```
>                     "setRequestHeader"in g && Z.forEach(s.toJSON(), (function(e, t) {
>                             g.setRequestHeader(t, e)
>                         }
> ```
> 
> Doesn't seem to prevent connecting

@mjbvz

**Comment by derekbekoe** [comment]
When using the Typescript in the web, some users see `Refused to set unsafe header "User-Agent"` in dev tools. This is because some browsers block setting this value. Technically, this should no longer be blocked by browsers. However today, Chrome still blocks it. See the notes in https://developer.mozilla.org/en-US/docs/Glossary/Forbidden_header_name.

Given the above and that this is not blocking, I'm closing this as a lower priority known issue.
